scripts/pnpm.ts

```diff
@@ -0,0 +1,5 @@
+import { readFileSync, writeFileSync } from 'fs'
+
+const pkg = JSON.parse(readFileSync('package.json', 'utf-8'))
+pkg.packageManager = `pnpm@${pkg.devDependencies.pnpm}`
+writeFileSync('package.json', `${JSON.stringify(pkg, null, 2)}\n`, 'utf-8')
```

Comment:
Use `readJson` and `writeJson` from `fs-extra` is more easily, right?
https://github.com/jprichardson/node-fs-extra/blob/master/docs/writeJson.md